src/PKSim.Core/Services/ParameterDefaultStateUpdater.cs

```diff
@@ -14,22 +15,44 @@ public class ParameterDefaultStateUpdater : IParameterDefaultStateUpdater
    {
       public void UpdateDefaultFor(ISpatialStructure spatialStructure)
       {
-         //TODO implement
+         var organism = spatialStructure.TopContainers.FindByName(Constants.ORGANISM);
+         if (organism == null) return;
+         updateDefaultStateForOriginDataParameters(organism);
       }
 
-
-      private void updateDefaultStateForOriginDataParameters(IContainer spatialStructureOrganism)
+      private void updateDefaultStateForOriginDataParameters(IContainer organism)
       {
-         new[] { CoreConstants.Parameter.AGE, Constants.Parameters.GESTATIONAL_AGE, CoreConstants.Parameter.HEIGHT }.Each(x =>
+         var parameterNames = new[]
          {
-            spatialStructureOrganism.Parameter(x).IsDefault = true;
-         });
-      }
+            CoreConstants.Parameter.AGE,
```

Comment:
The only 3 constant parameters that I could think of in Individual. We might need to  add more as we test it